client/js/components/month/month-day-card.js

```diff
@@ -1,31 +1,25 @@
 /** @jsx React.DOM */
 var React = require('react');
+var MonthDayEvent = require('./month-day-event');
 var Link = require('react-router-component').Link;
+var moment = require('moment');
 
 var MonthDayCard = React.createClass({
   render:function () {
-    var currentDate = this.props.dayData.date;
-    var dayPath = '/day/' + currentDate;
+    var weekNumber = Number(this.props.dayData.weekNumber);
+    var dayNumber = parseInt(moment(this.props.dayData.dayNumber, 'YYYYMMDD').format('DD'), 10);
```

Comment:
It was from when moment was causing problems and not converting to the correct day as expected, it will be removed in my corrections